package.json

```diff
@@ -26,8 +26,11 @@
     "test:lint:es": "eslint src test",
     "test:lint:ts": "tslint --project tsconfig.json 'src/**/*.{ts,tsx}' 'test/**/*.{ts,tsx}'",
     "test:types": "tsc --skipLibCheck --noEmit",
-    "test:unit": "jest",
-    "test:unit:coverage": "jest --coverage",
+    "test:integration": "jest integration",
+    "test:integration:coverage": "yarn test:integration --coverage",
```

Comment:
This one probably doesn't make much sense 🤔

Reply:
Yeah we can leave this one out for now.